e2e/wdio/platform/tests/radio-button-group.e2e-spec.ts

```diff
@@ -0,0 +1,436 @@
+    orientation,
+    winterValue,
+    radioButtonInlineCozyRadioGroup,
+    radioButtonWithDefaultPropertyValues,
+    radioButtonWithDisabledButtonAndValidationError,
+    januaryValue,
+    monthNames,
+    resetDisabledValidationGroupButtonIndex,
+    marchIndex,
+    radioButtonUsingFormGroupAndFormControl,
+    marchValue,
+    radioButtonInlineWithDefaultSelection,
+    radioButtonInlineCompact,
+    radioButtonWithDisabledItem,
+    radioButtonHaveValidationError,
+    resetHaveValidationErrorIndex,
+    radioButtonWithNoSelectionValue,
+    noneValue,
+    radioButtonWithFormGroupAndFormControl,
+    radioButtonWithPreSelection,
+    radioButtonWithGivenListOfStringValues,
+    radioButtonWithListOfSelectItemObjects,
+    resetListOfSelectitemObjectIndex,
+    seasonsNames,
+    radioButtonVertical,
+    resetVerticalRadioGroupButtonIndex,
+    resetQualificationFormButtonIndex,
+    radioButtonWithSelectItemsFormGroupAndFormControl,
+    radioButtonVerticalQualificationForm,
+    autumnIndex,
+    autumnValue,
+    radioButtonWithPreSelectionOnObjectDataPassed,
+    springIndex,
+    springValue,
+    radioButtonWithPreSelectionBasedOnControlValue,
+    platformRadioButtonWithSelectiemObject,
+    samsungValue,
+    radioButtonWithLookupKeyAndDisplayKey,
+    radioButtonWithDisabledContentAndFormcontrol,
+    radioButtonWithSomeDisabledButtons,
+    radioButtonCreatedFromList,
+    radioButtonDisabledWithSelectedItem,
+    radioButtonWithDisabledButtonsFromSelectitemObject,
+    radioButtonWithDisabledButtonsDrivenForm,
+    radioButtonWithIndividualButtonDisabled,
+    radioButtonDisabledCreatedFromList,
+    radioButtonDisabledCreatedFromSelectItem,
+    radioButtonWithDisabledItemsCreatedFromSelectItem,
+    submitDisabledRadioButtonWithValidationErrorIndex,
+    submitRadioButtonHaveValidationError,
+    submitRadioButtonWithListOfSelectItemObjects
+
+} from '../fixtures/testData/radio-button-group';
+
+describe('Radio button group  Test Suite', function() {
+    const radioButtonGroupPage: RadioButtonGroupPage = new RadioButtonGroupPage();
+    const {
+        formGroup, selectedValueLabel, preferredBrandLabel
+    } = radioButtonGroupPage;
+
+    beforeAll(() => {
+        radioButtonGroupPage.open();
+    }, 1);
+
+    afterEach(() => {
+        refreshPage();
+    });
+
+    it('Verify Radio button buttons can be aligned vertically and horizontally.', () => {
+        const groupButtons = elementArray(formGroup);
+        for (let i = 1; i < groupButtons.length; i++) {
+            expect(orientation).toContain(getAttributeByName(formGroup, 'aria-orientation', i));
+        }
+
+    });
+
+    it('Compact radio group with "default" state and with default selection', () => {
+        scrollIntoView(radioButtonGroupPage.radioButtonInputByName(radioButtonWithDefaultStateAndSelectionIndex));
+        click(radioButtonGroupPage.radioButtonLabelByName(radioButtonWithDefaultStateAndSelectionIndex));
```

Comment:
Please add the methods from the PO into the object destruction. Only ones that won't work are "open" and "checkRtlSwtch"